.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
We should adopt a larger `.gitignore`, e.g. https://github.com/github/gitignore/blob/master/Node.gitignore